server/index.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('../database/index.js')
+const transactions = require('../database/Transaction.js')
+const stocks = require('../database/Stock.js')
+const port = 3333;
+
+// app.use(express.static('public'));
+app.use(bodyParser.json());
+
+app.get('/:stockTicker', (req, res) => {
+    console.log('PARAMMMMSSSSSS', req.params);
+    stocks.getStock(req.params.stockTicker, (err, data) => {
+        if (err) {
+            res.json({message: 'what\'s a stock?'});
+        } else {
+            res.json(data)
+        }
+    });
+});
+
+app.post('/', (req, res) => {
+    
+    var transaction = {
+        stockName: req.body.stockName,
+        stockTicker: req.body.stockTicker,
+        currentPrice: req.body.currentPrice,
+        quantity: req.body.quantity
+    }
+
+    transactions.newTransaction(transaction, (err, data) => {
+        if (err) {
+            res.json({message: 'ooo, guess you missed that trade... hope it wasn\'t important!'});
```

Comment:
nice error handling!

1. could be useful to set status code to 500 or something
2. small trick: you can avoid having to escape the `'` by using double quotes instead `"ooo, guess you missed that trade... hope it wasn't important!"`

Reply:
thanks, and implemented!